service-config/build.gradle

```diff
@@ -0,0 +1,13 @@
+apply from: "$rootDir/gradle/libs.gradle"
+apply from: "$rootDir/gradle/publish.gradle"
+
+dependencies {
+    compile project(':http-clients')
```

Comment:
I am surprised that the dependency goes this way. I suspect http-clients should depend on on service-config instead?